src/app/compare/Compare.tsx

```diff
@@ -235,35 +221,49 @@ export default function Compare({ session }: { session: CompareSession }) {
     </div>
   );
 
+  const isShiftHeld = useShiftHeld();
   return (
     <Sheet onClose={cancel} header={header} allowClickThrough>
       <div className={styles.bucket} onPointerLeave={() => setHighlight(undefined)}>
         <div className={styles.statList}>
           <div className={styles.spacer} />
-          {allStats.map((stat) => (
-            <div
-              key={stat.id}
-              className={clsx(styles.statLabel, {
-                [styles.sortDesc]: stat.id === sortedHash && sortBetterFirst,
-                [styles.sortAsc]: stat.id === sortedHash && !sortBetterFirst,
-              })}
-              onPointerEnter={() => setHighlight(stat.id)}
-              onClick={() => changeSort(stat.id)}
-            >
-              {stat.displayProperties.hasIcon && (
-                <span title={stat.displayProperties.name}>
-                  <BungieImage src={stat.displayProperties.icon} />
-                </span>
-              )}
-              {stat.id in statLabels
-                ? t(statLabels[stat.id as StatHashes]!)
-                : stat.displayProperties.name}{' '}
-              {stat.id === sortedHash && (
-                <AppIcon icon={sortBetterFirst ? faAngleRight : faAngleLeft} />
-              )}
-              {stat.id === highlight && <div className={styles.highlightBar} />}
-            </div>
-          ))}
+          {allStats.map((stat) => {
+            const columnSort = columnSorts.find((c) => c.columnId === stat.id.toString());
+            return (
+              <div
+                key={stat.id}
+                className={clsx(
+                  styles.statLabel,
+                  columnSort
+                    ? columnSort.sort === SortDirection.ASC
+                      ? styles.sortDesc
+                      : styles.sortAsc
```

Comment:
This is a bit awkward but I'll clean it up later - the original styles were actually backwards between "ascending" and "descending".